src/cable/channel.cr

```diff
@@ -8,6 +8,19 @@ module Cable
       Cable::Channel::CHANNELS[self.name] = self
     end
 
+    # @override in after_subscribed macro
+    def run_after_subscribed_callbacks
+    end
+
+    # Run `block` after the subscription is created.
+    macro after_subscribed(*callbacks)
+      def run_after_subscribed_callbacks
+        {% for callback in callbacks %}
+          {{callback.id}}
+        {% end %}
+      end
+    end
```

Comment:
I never thought too much about callbacks, I loved how simple you implemented it! that's very nice